apps/astarte_appengine_api/test/support_v2/cases/database.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Astarte.Test.Cases.Database do
+  use ExUnit.CaseTemplate
+
+  using do
+    quote do
+      # Do nothing for now
+    end
+  end
```

Comment:
If this is not used, it can be removed, see this example from ExUnit doc: https://hexdocs.pm/ex_unit/ExUnit.CaseTemplate.html#module-example